docs/2023/CARE/Adding typesafety to teleicu middleware/index.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 1
+title: Introduction
+slug: /2023/CARE/Adding typesafety to teleicu middleware
+---
+
+
+# Author
+Aman shenoy
+
+# Contact Info
```

Comment:
add contact info
also do consider adding `mailto:` before the email and add https links for github and linkedin